mainwindow.ui

```diff
@@ -13,14 +13,193 @@
   <property name="windowTitle">
    <string>Jojibrush</string>
   </property>
-  <widget class="QWidget" name="centralWidget"/>
+  <widget class="QWidget" name="centralWidget">
```

Comment:
Can you extract this into its own .ui file plz? This is a good candidate for another QWidget

Reply:
Interesting, should try to do that

Reply:
This suggestion would be a whole lot more awesome if I knew what it meant 😅.  Are you talking about adding another `.ui` and calling it "centralwidget.ui"?  I thought each `.ui` represented its own fully seperate window